apps/wallet-dashboard/components/migration/migration-object-details-card/MigrationObjectDetailsCard.tsx

```diff
@@ -124,6 +138,8 @@ function UnlockConditionLabel({ groupKey, isTimelocked: isTimelocked }: UnlockCo
     const epochStartMs = currentEpochStartTimestampMs ?? '0';
     const currentDateMs = Date.now();
 
+    const originTimelockUnlockConditionMs =
```

Comment:
i would reverse the logic so you name this `estimatedTimelockUCMs` and add the 24h, and display , and I would explain why you are doing so, we need to add comments explaining the logic

Reply:
Also the logic is the othyer way around, we have to display in the table the estimated logic, so the unix + 24h with a `~`, and in the tooltip explain a bit more

Reply:
naming this `TimelockUnlockCondition` is innacurate since this applies to timelocks and expiration UCs